# HG changeset patch
# Parent 88d688d600459869f467351db6ed200bd45d287e
Use SO_REUSEADDR on the broadcast socket

diff --git a/lifx/broadcast.c b/lifx/broadcast.c
--- a/lifx/broadcast.c
+++ b/lifx/broadcast.c
@@ -286,6 +286,12 @@
     if (err) {
         goto error;
     }
+    err = evutil_make_listen_socket_reuseable(
+        lgtd_lifx_broadcast_endpoint.socket
+    );
+    if (err) {
+        goto error;
+    }
 
     err = evutil_make_socket_nonblocking(lgtd_lifx_broadcast_endpoint.socket);
     if (err == -1) {
